### PR TITLE
example/ssd/evaluate/eval_metric.py

### PR DESCRIPTION
**Pull request by sunrongda**
## Description ##
(Brief description on what this PR is about)
image
 this sort nothing to do  gets,after sort gets didn't change

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by abhinavs95** [comment]
Thank you for contributing @sunrongda 

@mxnet-label-bot add [Example, pr-awaiting-review]

**Comment by abhinavs95** [comment]
@szha @zhreshold Could you help review?

File: example/ssd/evaluate/eval_metric.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def iou(x, ys):
                 dets = pred[indices]
                 pred = np.delete(pred, indices, axis=0)
                 # sort by score, desceding
-                dets[dets[:,1].argsort()[::-1]]
+               # dets[dets[:,1].argsort()[::-1]]
```
Comment (zhreshold) [inline example/ssd/evaluate/eval_metric.py]: dets = dets[dets[:,1].argsort()[::-1]]

Comment (sunrongda) [inline example/ssd/evaluate/eval_metric.py]: dets = dets[dets[:,1].argsort()[::-1]]

**Comment by sunrongda** [comment]
@szha @zhreshold Could you help review?

**Comment by sunrongda** [comment]
> Could you please provide the change of MAP before and after the fix?
this is my 
image

this is no change map
image
this is after change map
image
As you can see, map changes a lot.

**Comment by sunrongda** [comment]
Hi, may I ask why this modification cannot be merged?

**Pull request merged**